File: src/main/java/cn/labzen/logger/meta/LoggerMeta.java
package cn.labzen.logger.meta;

import cn.labzen.meta.component.DeclaredComponent;

public class LoggerMeta implements DeclaredComponent {

  @Override
  public String mark() {
    return "Labzen-Logger";
  }

  @Override
  public String packageBased() {
    return "cn.labzen.logger";
  }

  @Override
  public String description() {
    return "基于SLF4j日志门户，封装Logback或Reload4j(Log4j2)提供增强的日志功能";
  }
}

File: src/main/kotlin/cn/labzen/logger/meta/LoggerMeta.kt
package cn.labzen.logger.meta

import cn.labzen.meta.component.LabzenComponent

class LoggerMeta : LabzenComponent {
  override fun mark(): String =
    "Labzen-Logger"

  override fun packageBased(): String =
    "cn.labzen.logger"

  override fun description(): String =
    "基于SLF4j日志门户，封装Logback或Log4j2提供增强的日志功能"
}
